frontend/landing_page/src/pages/AboutUs/index.js

```diff
@@ -30,91 +30,71 @@ function AboutUs() {
-              </li>
-              <li>
-                <strong>Tile Inspection:</strong>
-                <span> to quickly examine and validate coadded images, only available for members of the collaboration.</span>
-              </li>
-            </ul>
-            <p><span>For more information see the <a href="https://docs.linea.org.br" target="_blank">documentation</a> and <a href="https://scienceserver-dev.linea.org.br/tutorials" target="_blank">video tutorials</a>.</span></p>
-            <p><strong><em>Credits</em></strong></p>
-            <p><span>We would like to thank the contribution of the following people and organizations</span></p>
-            <ul>
-              <li><span>NCSA team for providing the infrastructure and some of the tools used by the Science Server</span></li>
-              <li><span>E. Bertin for the use of VisiOmatic and for helping in its installation</span></li>
-              <li><span>P. Fernique for helping in the integration of Aladin in the Science Server framework</span></li>
-              <li><span>Cristiano Singulani (Technical Lead of the DES Science Portal)&nbsp;</span></li>
-              <li><span>Glauber Costa (Technical Lead of the LIneA Science Server)</span></li>
-              <li><span>Current and past LIneA IT team members.</span></li>
-            </ul>
-            </div>
-          </Grid>
+            <Grid
+              item
+              md={7}
+              sm={10}
+              className={classes.grid}
+            >
+              <div>
+                <p><strong><em>About us</em></strong></p>
+                <p>LIneA - Laboratório Interinstitucional de e-Astronomia - is a multi-user laboratory operated by a non-profit organization (LIneA Association) with financial support predominantly from the Brazilian Ministry of Science, Technology, and Innovation. Our mission is to partner with the <a href="https://www.linea.org.br/inct#inct" target='_blank' rel='noopener'>INCT of e-Universe</a> to support the Brazilian astronomical community with computational infrastructure and expertise in big data analysis to provide technical conditions for participation in major astronomical surveys such as <em>Sloan Digital Sky Survey</em> <a href="https://www.sdss.org" target='_blank' rel='noopener'>(SDSS)</a>, <em>Dark Energy Survey</em> <a href="https://www.darkenergysurvey.org" target='_blank' rel='noopener'> (DES)</a>, and <em>Legacy Survey of Space and Time</em> <a href="https://rubinobservatory.org" target='_blank' rel='noopener'>(LSST)</a>. To learn more, check out the video <a href="https://youtu.be/jC-k85tfd0Y" target='_blank' rel='noopener'>"Get to Know LIneA"</a> on our <a href="https://www.youtube.com/@linea_org" target='_blank' rel='noopener'>YouTube channel</a> or browse our <a href="https://linea.org.br/" target='_blank' rel='noopener'>official website</a>.</p>
```

Comment:
... "Get to Know LIneA"</a> (talk in Brazilian Portuguese) on our <a href="https: ...